channels/models.py

```diff
@@ -82,7 +82,7 @@ def __str__(self):
         return self.title
 
 
-class FieldChannel(BaseChannel, TimestampedModel):
+class Channel(BaseChannel, TimestampedModel):
```

Comment:
I'm going to opt to leave this for now because I'm more afraid of the migration this will create than the others. I realize `BaseChannel` isn't actually used anywhere, so there are no instances of `BaseChannel` objects on their own to migrate, but if I took the fields from `BaseChannel` and put them on `Channel`, wouldn't that still trigger `makemigrations` to pick up that the fields have been removed and re-added? I'm afraid of data loss in that scenario, but maybe that concern is unfounded and Django is smart enough to realize we're just getting rid of the base class?

Reply:
Actually, scratch that, I just tried it and it doesn't generate a migration, so I will go ahead and do this.